modules/server_new/src/main/scala/observe/server/engine/EventResult.scala

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2016-2025 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package observe.engine
+
+import observe.server.SeqEvent
+
+sealed trait EventResult extends Product with Serializable
+
+object EventResult:
+  enum Outcome:
+    case Ok, Failure
+
+  case class UserCommandResponse[F[_]](
+    userEvent: UserEvent[F],
+    outcome:   Outcome,
+    ud:        Option[SeqEvent]
```

Comment:
ud?

Reply:
I'd love to know what `ud` means so that we can rename it.

Reply:
User something?